services/user/Workshop/service/src/lib.rs

```diff
@@ -0,0 +1,138 @@
+#[psibase::service]
+#[allow(non_snake_case)]
+pub mod service {
+    use async_graphql::*;
+    use psibase::*;
+    use serde::{Deserialize, Serialize};
+
+    /// Holds metadata for a registered app
+    #[table(name = "AppMetadataTable", index = 0)]
+    #[derive(Debug, Clone, Fracpack, ToSchema, Serialize, Deserialize, SimpleObject)]
+    pub struct AppMetadata {
```

Comment:
These weren't in the github issue (until right now because I just added them), but could you also add 
* `tags: List<String>` - Devs can add up to 3 tags that can further be used to categorize the application
* `created: TimePointSec` - This can be set with getting `currentBlock().time` from the `Transact` service. Tracks the unix time in seconds that the app was registered.